src/test/regress/expected/master_evaluation.out

```diff
@@ -0,0 +1,127 @@
+CREATE SCHEMA master_evaluation;
```

Comment:
we seem to have specific tests for local execution master  evaluation for modifications:  https://github.com/citusdata/citus/blob/2f0ef8bc36d84283cbac055dd1b00028b2726988/src/test/regress/expected/local_shard_execution.out#L1339-L1415